whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapObjectMapperTest.java

```diff
@@ -223,13 +238,16 @@ public void asBlock() {
         assertThat(result.getCountry(), is(nullValue()));
 
         final List<Entity> entities = result.getEntitySearchResults();
-        assertThat(entities, hasSize(2));
+        assertThat(entities, hasSize(3));
+
         assertThat(entities.get(0).getHandle(), is("AP1-TEST"));
         assertThat(entities.get(0).getRoles(), containsInAnyOrder(Role.TECHNICAL, Role.ADMINISTRATIVE));
         assertThat(entities.get(0).getVCardArray(), is(nullValue()));
-        assertThat(entities.get(1).getHandle(), is("UPD-MNT"));
-        assertThat(entities.get(1).getRoles().get(0), is(Role.REGISTRANT));
-        assertThat(entities.get(1).getVCardArray(), is(nullValue()));
+        assertThat(entities.get(1).getHandle(), is("ORG-NCC1-RIPE"));
```

Comment:
Are the entities always in a deterministic order?

Reply:
Yes, We are using treemap RdapObjectMapper - line 307, although RFC does not enforce the natural order for entities

Reply:
I was thinking more about the tests running in a deterministic way as we've had some tests here and there that sometimes fail because of indexes being shuffled but in this case great! :)